components/posts/ContentRenderer.tsx

```
@@ -62,55 +54,44 @@ export const ContentRenderer = ({ item, index }: ContentRendererProps) => {
-          className={`${baseClasses} text-white/80 leading-relaxed mb-6 text-lg`}
-        >
+        <p className={`${baseClasses} mb-6 text-lg leading-relaxed text-white/80`}>
           {item.content}
         </p>
       );
 
     case "markdown":
       return (
-        <div className={`${baseClasses} prose prose-invert max-w-none mb-6`}>
+        <div className={`${baseClasses} prose prose-invert mb-6 max-w-none`}>
           <div dangerouslySetInnerHTML={{ __html: item.content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)